x-pack/plugins/observability/public/pages/slos/components/slo_list.tsx

```diff
@@ -42,16 +43,21 @@ export function SloList() {
   useEffect(() => {
     if (shouldReload) {
       setShouldReload(false);
+      setIsCloning(false);
       setIsDeleting(false);
     }
   }, [shouldReload]);
 
-  const handleDeleted = () => {
-    setShouldReload(true);
+  const handleCloning = (cloning: boolean) => {
+    setIsCloning(cloning);
+  };
+
+  const handleDeleting = (deleting: boolean) => {
+    setIsDeleting(deleting);
```

Comment:
Indeed, this was exactly what I thought as well. Tbh, this is going to change if and when we move to React Query: it will have less prop plumbing because we can just listen for the api interaction finishing inside the list component.